src/uct/tcp/tcp_sockcm_ep.h

```diff
@@ -59,6 +59,12 @@ uct_tcp_sockcm_t *uct_tcp_sockcm_ep_get_cm(uct_tcp_sockcm_ep_t *cep)
                             super.iface);
 }
 
+static UCS_F_ALWAYS_INLINE
+ucs_log_level_t uct_tcp_sockcm_get_failure_log(uct_tcp_sockcm_ep_t *cep)
```

Comment:
`uct_tcp_sockcm_ep_get_fail_log_lvl`

Reply:
BTW do we have any base parameter to configure both rdmacm and tcp?

Reply:
no, because the tcp_sockcm and rdmacm_cm cm_config_tables are set on their respective components, but there is no separate component for base_cm.